code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1033,7 +1033,7 @@
 	if(usr.incapacitated() || usr.stat > CONSCIOUS)
 		return
 
-	var/obj/item/weapon/implant/reagent_generator/roiz/rimplant
+	var/obj/item/weapon/implant/reagent_generator/pumila_apple/rimplant
```

Comment:
Given the organ thing, do you even need this anymore at all?

Reply:
Probably not. Let me remove it.